### PR TITLE
Update local setup docs

### PR DESCRIPTION
**Pull request by theskumar**
- Remove dependency on python-venv and python-pip
- update how to run test

### CODE REVIEW

File: docs/developers/localdevelscratch.md [hunk 1]
```diff
@@ -366,16 +358,16 @@ Now you should be able to access the sites on [http://hypha.test/](http://hypha.
...
 ```text
-$ python manage.py test --settings=hypha.settings.test
+$ make test
 ```
 
 If you need to rerun the tests several times this will speed them up considerably.
 
 ```text
-$ python manage.py test --parallel --keepdb --settings=hypha.settings.test
+$ py.test --reuse-db
```
Comment (frjo) [inline docs/developers/localdevelscratch.md]: Looks like `make test` runs `make py-test` and that command always use "--reuse-db".

Comment (frjo) [reply docs/developers/localdevelscratch.md]: And I think it is recommended to use `pytest` instead of the old `py.test`.

Comment (theskumar) [reply docs/developers/localdevelscratch.md]: I'll look at them, and update. :)

Comment (theskumar) [reply docs/developers/localdevelscratch.md]: Update the text to be more descriptive. Thanks for the feedback.

**Pull request merged**